redisinsight/api/src/modules/cli/services/cli-analytics/cli-analytics.service.spec.ts

```diff
@@ -42,21 +42,21 @@ describe('CliAnalyticsService', () => {
 
   describe('sendCliClientCreatedEvent', () => {
     it('should emit CliClientCreated event', () => {
-      service.sendCliClientCreatedEvent(instanceId, { data: 'Some data' });
+      service.sendClientCreatedEvent(instanceId, 'cli', { data: 'Some data' });
```

Comment:
use `AppTool.CLI` instead of string 'cli'

Reply:
thanks. done